PinpointKit/PinpointKit/Sources/Core/SystemLogCollector.swift

```diff
@@ -22,7 +22,19 @@ open class SystemLogCollector: LogCollector {
     
     private let logger: ASLLogger
     
-    public init(loggingType: LoggingType = .application) {
+    /**
+     Creates a new system logger.
+     
+     - parameter loggingType: Specifies the type of logs to collect.
+     
+     - warning: This initializer returns `nil` on iOS 10.0+. When running on iOS 10.0+, ASL is superseded by unified logging, for which there are no APIs to search or read log messages.
```

Comment:
Yeah, and we should definitely update other areas of the README in the areas where we talk about logging. I’ll do that in a separate PR if that’s okay.